code/__DEFINES/subsystems.dm

```diff
@@ -113,6 +113,7 @@
 #define INIT_ORDER_VIS 80
 #define INIT_ORDER_ACHIEVEMENTS 77
 #define INIT_ORDER_RESEARCH 75
+#define INIT_ORDER_SECURITY_LEVEL 75
```

Comment:
Does the security level subsystem depend on being before or after another subsystem? At a glance, I don't see any dependencies on any other subsystems.

If there are no dependencies, then this should just use the default place in the init order.

Reply:
Changed to default place.